docs/modules/user-guide/partials/proc_updating-the-devfile-library-schema.adoc

```diff
@@ -0,0 +1,35 @@
+[id="proc_updating-the-devfile-library-schema_{context}"]
+= Updating the devfile library schema
+
+[role="_abstract"]
+To ensure that you develop with the latest code from devfile properties, update the devfile library schema to the latest version.
+
+.Procedure
+
+. Add execute permissions for everyone on this script:
++
+[source,terminal]
+----
+$ chmod a+x r./scripts/updateApi.sh
+----
+
+. To fetch the latest `github.com/devfile/api` Go module and to update the schema saved under `pkg/devfile/parser/data`, enter:
++
+[source, terminal]
+----
+ $ ./scripts/updateApi.sh
+----
+
+. Update the devile schema version by running:
++
+[source, terminal]
+----
+$ ./updateAPI.sh 2.x.x
+----
+
+**Note:** Running the script this way, with no arguments, updates the devfile to the latest version.
```

Comment:
I'm confused here. The previous command contains the argument `2.x.x` and so saying `Running the script in this way` doesn't make sense to me. Maybe better to say `If you do not specify a version number as a command line argument, the script will update the devfile to the latest version.` Or something along those lines.